src/veil/ringct/anonwallet.cpp

```diff
@@ -4684,8 +4788,35 @@ bool AnonWallet::AddKeyToParent(const CKey& keySharedSecret)
     return fSuccess;
 }
 
+void AnonWallet::GetAllScanKeys(std::vector<CStealthAddress>& vStealthAddresses) {
+    LOCK(pwalletParent->cs_wallet);
+    int i = 0;
+    std::set<std::string> seenAddresses;
+
+    for (auto stealth: mapStealthDestinations) {
```

Comment:
```suggestion
    for (const auto& stealth: mapStealthDestinations) {
```
if these are usable as const references.